templates/set_tx_fee_surge_factor.cdc

```diff
@@ -0,0 +1,14 @@
+import FlowFees from 0xf919ee77447b7497
```

Comment:
this is coming from: https://github.com/onflow/flow-core-contracts/blob/master/transactions/FlowServiceAccount/set_tx_fee_surge_factor.cdc